content/en/docs/Tasks/creating-an-index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Creating an Index of operator bundles"
+weight: 3
+description: >
+  Once you have a collection of bundle/s, you can add them to an Index of bundles.
```

Comment:
A collection is always going to be of bundles no?
```suggestion
  Once you have a collection of bundles, you can add them to an Index of bundles.
```
And similar to the other descriptions e.g
- `Create an operator bundle using the operator manifests` 
- `List operators from a collection of catalogs`
can we reformat this to be similar to an actionable statement
```suggestion
  Add a collection of bundles to an Index
```